.github/workflows/workflow.yml

```diff
@@ -23,8 +23,6 @@ jobs:
           python-version: ${{ matrix.python-version }}
       - name: Build explainability-core
         uses: ./.github/actions/build-core
-      - name: Build arrow-converter
-        uses: ./.github/actions/build-arrow-converter
```

Comment:
Same change should be applied to [`publish.yml` workflow](https://github.com/trustyai-explainability/trustyai-explainability-python/blob/FAI-855/.github/workflows/publish.yml#L15-L16)